baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/LoggerEnclosingClass.java

```diff
@@ -83,8 +83,16 @@ public Description matchVariable(VariableTree tree, VisitorState state) {
         return buildDescription(classArgument)
                 .addFix(fix.replace(
                         classArgument,
-                        SuggestedFixes.prettyType(state, fix, enclosingClassSymbol.type) + ".class")
+                        enclosingClassSymbol.name + ".class")
                         .build())
                 .build();
     }
+
+    private static Symbol.ClassSymbol enclosingConcreteClass(Symbol.ClassSymbol input) {
+        Symbol.ClassSymbol current = input;
+        while (current != null && current.isAnonymous()) {
+            current = ASTHelpers.enclosingClass(current);
```

Comment:
I haven't encountered this in the wild (it's an odd thing to do) but fixed it while I'm here.